sds/src/scanner/mod.rs

```diff
@@ -73,6 +73,19 @@ where
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
This can be made `Copy` since it's trivial to copy, so you don't have to explicitly call `.clone()` on it (which is done later in this file)